testdata/configroot/scopes/global/subjects/global/rules.yml

```diff
@@ -32,4 +32,3 @@ rules:
       - descriptorName: RequestCount
         maxAmount: 5000
         expiration: 1s
```

Comment:
nit: leave newline at end of file

Reply:
so, I didn't remove that... and I can't convince git to put it back. awesome.

Reply:
ok. i think i tricked into the right thing now.